### DISCUSSION TITLE
How to set QR code to infinite repeat ? (photo)

### DISCUSSION THREAD

**Discussion by akox80** [gopro/labs#1149] [Q&A]
Hi,
i'm using the GoPro12 as a camera trap to record a few cryptic animals in my region. I'm using the QR Control App to set the motion parameters. When i slide the repeat function on, the motion detection only repeats once. Is there a way to set the camera to repeat the motion detection function infinitely?
thx, Jano

**Comment by dnewman-gpsw** [comment]
The old motion detection was not designed for photos.  The QRControl motion detection is the same as https://gopro.github.io/labs/control/motion/  which clearly states it is video only.

Fortunately new camera have some new features
![image](https://github.com/user-attachments/assets/048286f3-7a4c-4100-9c8e-9e9d97878683)

The uses the system variable 'm' which is a percentage of pixels in motion, so the above command (when in photo mode) will take an image is more that 2% of the pixel are changing.

**Reply by akox80** [reply]
Thank you! 
Jano
